daml/Marketplace/Custody/Model.daml

```diff
@@ -15,29 +16,63 @@ template SettlementInfo
     operator : Party
     party : Party
     ownAccount : Optional Account
-    cashAccount : Optional Account
-    securitiesAccount : Optional Account
+    omnibusCashAccount : Optional Account
```

Comment:
@georg-da can I regard omnibus accounts the same as settlement account or is it more like a backup account?
I can see that in Hierarchical.getAccount we will first do a lookup in cash/security accounts and if we can't find an account provide by the specify party we then look at omnibus account does this implies that omni bus account is a backup account / second choice?

Reply:
The omnibus account is the account for all holdings that don't (need to) have a segregated account. So you could set up segregated accounts for some investors, and manage others in the omnibus account. Or manage all client holdings segregated, but hold your own positions in the omnibus. It's really a catch-all account that's not tied to a specific party, how this is going to be used depends on a given operating model.